src/App.js

```diff
@@ -1,13 +1,13 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect } from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import Navigation from './component/Navigation.js';
-import LikeList from './component/LikeList.js';
-import MovieList from './component/MovieList.js';
-import BlockList from './component/BlockList.js'
-import Home from './component/Home.js'
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
you don't have to import "bootstrap/dist/css/bootstrap.min.css" in every script, I already import in index.js